apps/meteor/client/views/admin/apps/AppDetailsPage.tsx

```diff
@@ -76,15 +76,15 @@ const AppDetailsPage: FC<{ id: string }> = function AppDetailsPage({ id }) {
 
 	const { isSelectedDetails, isSelectedLogs, isSelectedSettings } = selectedTab;
 
-	const isSettingsTabSelected = Boolean(settings && Object.values(settings).length && isSelectedSettings);
-	const isSettingsTabEnabled = Boolean(settings && Object.values(settings).length && installed);
-
 	const isDetailsTabSelected = isSelectedDetails;
 	const areApisVisible = Boolean(isSelectedDetails && !!showApis);
 
 	const isLogsTabSelected = isSelectedLogs;
 	const isLogsTabEnabled = Boolean(installed);
 
+	const isSettingsTabSelected = Boolean(settings && Object.values(settings).length && isSelectedSettings);
```

Comment:
I suggest putting isSelectedSettings in the front of the condition, because in this case if the settings aren't selected don't need to verify if the settings were filled.

Reply:
Solved